packages/wdio-cli/src/setup.js

```diff
@@ -5,47 +5,51 @@ import inquirer from 'inquirer'
 import yarnInstall from 'yarn-install'
 
 import { CONFIG_HELPER_INTRO, CONFIG_HELPER_SUCCESS_MESSAGE, QUESTIONNAIRE } from './config'
+import { getNpmPackageName, getPackageName } from './utils'
+
+export default async function setup (exit = true) {
+    try {
+        console.log(CONFIG_HELPER_INTRO) // eslint-disable-line no-console
+        const answers = await inquirer.prompt(QUESTIONNAIRE)
+        const packagesToInstall = [
+            getNpmPackageName(answers.runner),
+            getNpmPackageName(answers.framework),
+            ...answers.reporters.map(getNpmPackageName),
+            ...answers.services.map(getNpmPackageName)
+        ]
 
-export default function setup () {
-    console.log(CONFIG_HELPER_INTRO) // eslint-disable-line no-console
-    inquirer.prompt(QUESTIONNAIRE).then((answers) => {
-        let packagesToInstall = []
-        if (answers.installRunner) {
-            packagesToInstall.push(`@wdio/${answers.runner}-runner`)
-        }
-        if (answers.installFramework) {
-            packagesToInstall.push(`@wdio/${answers.framework}-framework`)
-        }
-        if (answers.installReporter) {
-            packagesToInstall = packagesToInstall.concat(answers.reporters)
-        }
-        if (answers.installServices) {
-            packagesToInstall = packagesToInstall.concat(answers.services)
-            if (answers.services.includes('wdio-chromedriver-service')) {
-                packagesToInstall.push('chromedriver')
```

Comment:
I can't find a place where we adding chromedriver in case wdio-chromedriver-service was chosen

Reply:
Hm, must've gotten deleted when I resolved some conflicts. I'll add it again.

Reply:
Is this correct? 6b30654

Reply:
Yes, thank you